tardis/plasma/properties/nlte.py

```diff
@@ -103,3 +105,106 @@ def filter_with_helium_index(atomic_number, ion_number, partition_function,
             columns=['ionization_energy'])
         return partition_function_index, ionization_data_index,\
                partition_function, ionization_data
+
+class HeliumNumericalNLTE(ProcessingPlasmaProperty):
+    outputs = ('helium_population',)
+    '''
+    IMPORTANT: This particular property requires a specific numerical NLTE
+    solver and a specific atomic dataset (neither of which are distributed
+    with Tardis) to work.
+    '''
+    def calculate(self, ion_number_density, electron_densities, t_electrons, w,
+        lines, j_blues, levels, level_boltzmann_factor, t_rad,
+        zeta_data, g_electron, delta, partition_function, ionization_data,
+        beta_rad, g):
+        print 'Performing numerical NLTE calculations.'
+        if len(j_blues)==0:
+            return None
+        heating_rate_data = np.loadtxt(
+            self.plasma_parent.heating_rate_data_file, unpack=True)
+        #Outputting data required by SH module
+        for zone, _ in enumerate(electron_densities):
+            with open('He_NLTE_Files/shellconditions_%d.txt' %zone, 'w') as \
```

Comment:
to format strings we normally try to use new style formatting: `"I would like to buy {0} apples"`.format(5)